contract-examples/test/ExampleRewardManager.ts

```diff
@@ -0,0 +1,154 @@
+// (c) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import {
+  BigNumber,
+  Contract,
+  ContractFactory,
+} from "ethers"
+import { ethers } from "hardhat"
+import ts = require("typescript");
+
+// make sure this is always an admin for minter precompile
```

Comment:
should this be reward manager precompile?

Reply:
yup changed it